templates/worker/01-worker-kubelet/on-prem/units/kubelet.service.yaml

```diff
@@ -26,7 +26,7 @@ contents: |
         --container-runtime=remote \
         --container-runtime-endpoint=/var/run/crio/crio.sock \
         --runtime-cgroups=/system.slice/crio.service \
-        --node-labels=node-role.kubernetes.io/worker,node.openshift.io/os_id=${ID} \
+        --node-labels=node-role.kubernetes.io/worker,node.openshift.io/os_id=${ID},${LABELS} \
```

Comment:
Maybe instead of `LABELS`, we can consider using something more specific. 

Also, would you have an example of something injecting those labels?

Reply:
hmm, have better naming? I mean i can set REMOTE_WORKER_LABEL but i assumed we can use something generic

Reply:
I like the `ADDITIONAL_CUSTOM_LABELS` you have now, I just thought `LABELS` were too generic and could be accidentally provided. Thanks!